python_modules/dagster/dagster/_core/host_representation/external_data.py

```diff
@@ -479,6 +482,9 @@ def __new__(
         timezone: Optional[str],
         fmt: str,
         end_offset: int,
+        minute_offset: int,
+        hour_offset: int,
+        day_offset: Optional[int],
```

Comment:
adding non optional entries to existing `@whitelist_for_serdes` causes loading old entries to break